ui/packages/app/src/components/Gallery/Gallery.svelte

```diff
@@ -189,11 +189,19 @@
 							class="gallery-item group"
 							on:click={() => (selected_image = can_zoom ? i : selected_image)}
 						>
-							<img
-								alt=""
-								class="h-full w-full overflow-hidden object-contain"
-								src={typeof image === "string" ? image : image.data}
-							/>
```

Comment:
We don't need to duplicate this block and wrap it in an `if`, we can just fall back to an empty string:

```svelte
<img
  alt={caption || ""}
  class="h-full w-full overflow-hidden object-contain"
  src={typeof image === "string" ? image : image.data}
/>
```

Reply:
cool thanks ♥